sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveResolutionSuite.scala

```diff
@@ -38,6 +38,15 @@ class HiveResolutionSuite extends HiveComparisonTest {
     sql("SELECT a[0].A.A from nested").queryExecution.analyzed
   }
 
+  test("SPARK-5278: check ambiguous reference to fields") {
+    jsonRDD(sparkContext.makeRDD(
+      """{"a": [{"b": 1, "B": 2}]}""" :: Nil)).registerTempTable("nested")
+    val exception = intercept[RuntimeException] {
+      println(sql("SELECT a[0].b from nested").queryExecution.analyzed)
```

Comment:
Can you add a comment at here to explain what we are expecting? Also, you can remove `println`.